src/pages/index.tsx

```diff
@@ -23,26 +30,36 @@ interface IndexPageQueryProps {
   officeImages: {
     nodes: OfficeImage[];
   };
+  servicesHeader: ContentfulSectionHeader;
+  careerHeader: ContentfulSectionHeader;
+  servicesTeaser: ContentfulTeaser;
+  blogHeader: ContentfulSectionHeader;
 }
 
-const IndexPage = (props: PageProps<IndexPageQueryProps>) => {
-  const jobPositions = props.data.allContentfulVacancy.nodes;
-  const blogPosts = props.data.allContentfulBlogPost.nodes;
-  const officeImages = props.data.officeImages.nodes.reduce((memo, image) => {
+const IndexPage = ({ data, location }: PageProps<IndexPageQueryProps>) => {
+  const jobPositions = data.allContentfulVacancy.nodes;
+  const blogPosts = data.allContentfulBlogPost.nodes;
+  const officeImages = data.officeImages.nodes.reduce((memo, image) => {
     memo[image.relativePath] = image;
     return memo;
   }, {});
 
   return (
     <>
-      <SEO title="Satellytes" location={props.location} rssLink />
+      <SEO title={data.contentfulPage.title} location={location} rssLink />
 
       <StructuredOrganizationData />
 
       <Landingpage
+        title={data.contentfulPage.title}
+        description={data.contentfulPage.description?.description as string}
         officeImages={officeImages}
         positions={jobPositions}
         posts={blogPosts}
+        serviceHeader={data.servicesHeader}
+        serviceTeaser={data.servicesTeaser.gridItems}
+        careerHeader={data.careerHeader}
+        blogHeader={data.blogHeader}
```

Comment:
at least we have the presentational separation present not to mix data and the actual display of that data. Things would look much worse with the many data involved now.